style/style.css

```diff
@@ -317,52 +451,118 @@ body {
 }
 
 .button-section {
-  width: 32.7rem;
+  width: 100%;
   height: 4.8rem;
   display: flex;
-  background: var(--color-sccent);
-  justify-content: center;
-  align-items: center;
-  margin-top: 2.2rem;
 }
 
 .card-button {
   text-decoration: none;
+  text-align: center;
+  padding: 1.2rem 0;
+  width: 100%;
   font-size: 1.7rem;
   font-weight: 700;
+  background-color: var(--color-sccent);
   cursor: pointer;
 }
 
 .white-font {
   color: var(--color-white);
 }
 
+@media screen and (min-width: 768px) {
+  .work-section-container {
+    margin: auto;
+    padding: 12rem 14.35rem;
+  }
+
+  .title-card {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .title-separator {
+    width: 100%;
+    align-self: center;
+  }
+
+  .image {
+    width: 100%;
+  }
+
+  .card-holder {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 2.4rem;
+    row-gap: 3.7rem;
+    margin: auto;
+  }
+
+  .work-card {
+    width: 100%;
+  }
+
+  .card-button:hover {
+    color: var(--color-sccent);
+    background-color: var(--color-secondary);
+    border: 1px solid var(--color-sccent);
+  }
+}
```

Comment:
- [ ] By setting the min-width of 768px of your media query, part of this section is cut out of the screen. Kindly fix the margin and padding of this section.
- [ ] The image width of the featured project should take a more significant portion of the screen size. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/171678706-7b22c22f-44b8-4975-86b3-d0bdd769e868.png)
![image](https://user-images.githubusercontent.com/61919617/171678903-ec8dba25-42f0-4a03-8c47-6f144b618939.png)